src/hiero_sdk_python/tokens/token_update_nfts_transaction.py

```
@@ -5,35 +5,40 @@
 Provides TokenUpdateNftsTransaction, a subclass of Transaction for updating
 metadata of non-fungible tokens (NFTs) on the Hedera network via HTS.
 """
+
 from typing import List, Optional
```

Ruff (0.14.14)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)